### ISSUE TITLE
Unsorted/Ungrouped option to overlap with all other sort modes (Feature/Option)

### ISSUE THREAD

**Issue by HyperLemons** [etternagame/etterna#1011]
A feature that would be very useful would be having all charts be unsorted instead of the default sort by chart packs. This would not be a sort mode with all the others, but to go along with them as a separate option.

Example: Enabling Grouping Off and then changing to "Overall" in the Sort Menu would make for sorting all charts by difficulty, which can be really resourceful for someone not wanting to look through all of their packs to play by difficulties, by instead have them all there at once.

A comparison ONLY for the sake of the understanding this feature better would be in Quaver, where there is a "Group By" option for having unsorted and sorting by playlists.

![image](https://user-images.githubusercontent.com/36387099/126228310-f4fcf923-eb53-447c-9c63-dac9b58be4aa.png)

Similarly in osu!mania (osu! does have way more grouping options however)
![image](https://user-images.githubusercontent.com/36387099/126227192-9f887b84-c484-45a7-9f9c-262a5e92b7c6.png)

Basically, it would leave for an option to be enabled. The "Grouping" option menu could enabled in certain combo like enabling the Sort Menu, then pressing UP DOWN UP DOWN once more to enable the "Grouping" menu to enable or disable.

Ideally looking something like this:

![image](https://user-images.githubusercontent.com/36387099/126232747-e2a7954c-c9fa-4c98-8d65-65a94a1b49cc.png)

There could be other ways to go about enabling it, but I personally think that this is the best and simplest way. (if possible)

Hopefully you can take this into consideration. :)

**Comment by casey2069** [comment]
This is already theoretically possible, just a bit fiddly, as the user has to either take the hit of loading more songs or maintain 2 separate caches.
Below is an overall sort of ungrouped charts.
![pic-full-220407-0955-00](https://user-images.githubusercontent.com/23276080/162258516-d67efa77-9a1c-4a88-a666-3421762c63e7.png)
